src/OmniSharp.Stdio.Driver/app.config

```diff
@@ -51,6 +51,10 @@
                 <assemblyIdentity name="System.Composition.TypedParts" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
                 <bindingRedirect oldVersion="0.0.0.0-1.0.31.0" newVersion="1.0.31.0"/>
             </dependentAssembly>
+            <dependentAssembly>
```

Comment:
this file (and other binding redirects files) have MsBuild redirects to 15.1.0.0, I think they should be updated too?

Reply:
Tried updating this and binding failed; opened assemblies from msbuild 16 and the version is still 15.1...

Reply:
oh.. of course it is... 🤯